src/repositories/userRepository.js

```diff
@@ -0,0 +1,23 @@
+const User = require('../models/userModel');
+
+const createUser = (userData) => User.create(userData, {});
+
```

Comment:
What is the purpose of the second parameter of the create method? 

Here is an example from the documentation about creating a user:  

![image](https://github.com/user-attachments/assets/d2cd28c5-a559-4de9-89c2-e1dfe89e3416)